app/models/language_skill.rb

```diff
@@ -1,6 +1,11 @@
 class LanguageSkill < ApplicationRecord
   belongs_to :languageable, polymorphic: true, optional: true
 
+  scope :native, -> { where(level: 'native_speaker') }
+  scope :german, -> { where(language: 'DE') }
+  scope :native_language, -> { native.german.first || native.first || LanguageSkill.new }
```

Comment:
This should be a normal method, scopes are expected to return an `ActiveRelation` in order to be chainable.